apps/token-wrapper/buidler.config.js

```diff
@@ -0,0 +1,64 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+const hooks = require('./scripts/buidler-hooks')
+
+usePlugin('@aragon/buidler-aragon')
```

Comment:
Since this plugin is not really in active development anymore, what do you think about removing any configuration or dependencies that rely on it? Or was this still essential to your workflow?